SysKit.XCellKit/Tables/SpreadsheetTable.cs

```diff
@@ -168,7 +170,7 @@ public Table GetTableDefinition(int id, int columnIndex, int rowIndex)
                 }
                 i++;
             }
-            TableStyleInfo tableStyle = new TableStyleInfo() { Name = "TableStyleLight9", ShowFirstColumn = false, ShowLastColumn = false, ShowRowStripes = true, ShowColumnStripes = false, };
+            TableStyleInfo tableStyle = new TableStyleInfo() { Name = StyleName, ShowFirstColumn = false, ShowLastColumn = false, ShowRowStripes = false, ShowColumnStripes = false, };
```

Comment:
Maybe have default value for the StyleName when it is not provided by the user of the library? 
Will this break backward compatibility? Because you now have to provide StyleName for the table?

Reply:
The style name is by default TableStyleLight9 as it was beforehand. The value is set in the 37th line inside the class constructor. The only difference now is that that value can be overridden from the outside.